src/debugAdapter/goDebug.ts

```diff
@@ -754,13 +761,6 @@ class GoDebugSession extends LoggingDebugSession {
 		log('InitializeResponse');
 	}
 
-	protected findPathSeperator(filePath: string) {
-		if (/^(\w:[\\/]|\\\\)/.test(filePath)) {
-			return '\\';
-		}
-		return filePath.includes('/') ? '/' : '\\';
```

Comment:
Is there a possibility that the path has both / and \? The new change selects / in that case while the old findPathSeparator picks \  if it matches the above pattern (in line 758). Wonder if that would be potentially a problem.